samples/azure/native-integration-asq/ASQN_10/Receiver/Program.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.Azure.Transports.WindowsAzureStorageQueues;
+
+class Program
+{
+    static async Task Main()
+    {
+        var endpointName = "native-integration-asq";
+        Console.Title = endpointName;
+
+        var endpointConfiguration = new EndpointConfiguration(endpointName);
+        endpointConfiguration.UseSerialization<NewtonsoftSerializer>();
+        endpointConfiguration.EnableInstallers();
+
+        var transport = new AzureStorageQueueTransport("UseDevelopmentStorage=true");
+        // TODO: ASQ transport is not yet ready, Core v8 work is still in progress
+        // transport.DisablePublishing();
+        // transport.DelayedDelivery().DisableDelayedDelivery();
```

Comment:
To disable native delayed deliveries use the following constructor overload:

```
var transport = new AzureStorageQueueTransport("UseDevelopmentStorage=true", useNativeDelayedDeliveries: false );
```

Reply:
Thank you. Too bad `transport.DelayedDelivery().DisableDelayedDelivery();` does not point in the right direction. Someone might overlook the constructor overload for the transport just like I did.

Reply:
@mauroservienti, this doesn't seem to work as expected. The `RecoverabilityComponent` still throws.

<details>
<summary>Exception</summary>

System.Exception
  HResult=0x80131500
  Message=Delayed retries are not supported when the transport does not support delayed delivery.
  Source=NServiceBus.Core
  StackTrace:
   at NServiceBus.RecoverabilityComponent.GetDelayedRetryConfig() in /_/src/NServiceBus.Core/Recoverability/RecoverabilityComponent.cs:line 140
   at NServiceBus.RecoverabilityComponent.Initialize(Configuration receiveConfiguration, Configuration hostingConfiguration, TransportSeam transportSeam) in /_/src/NServiceBus.Core/Recoverability/RecoverabilityComponent.cs:line 56
   at NServiceBus.EndpointCreator.Configure() in /_/src/NServiceBus.Core/EndpointCreator.cs:line 65
   at NServiceBus.EndpointCreator.Create(SettingsHolder settings, Configuration hostingConfiguration) in /_/src/NServiceBus.Core/EndpointCreator.cs:line 28
   at NServiceBus.HostCreator.<CreateWithInternallyManagedContainer>d__1.MoveNext() in /_/src/NServiceBus.Core/Hosting/HostCreator.cs:line 57
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable`1.ConfiguredTaskAwaiter.GetResult()
   at NServiceBus.Endpoint.<Start>d__1.MoveNext() in /_/src/NServiceBus.Core/Endpoint.cs:line 29
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable`1.ConfiguredTaskAwaiter.GetResult()
   at Program.<Main>d__0.MoveNext() in C:\github\nsb\docs.particular.net\samples\azure\native-integration-asq\ASQN_10\Receiver\Program.cs:line 39
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Program.<Main>()

  This exception was originally thrown at this call stack:
    NServiceBus.RecoverabilityComponent.GetDelayedRetryConfig() in RecoverabilityComponent.cs
    NServiceBus.RecoverabilityComponent.Initialize(NServiceBus.ReceiveComponent.Configuration, NServiceBus.HostingComponent.Configuration, NServiceBus.TransportSeam) in RecoverabilityComponent.cs
    NServiceBus.EndpointCreator.Configure() in EndpointCreator.cs
    NServiceBus.EndpointCreator.Create(NServiceBus.Settings.SettingsHolder, NServiceBus.HostingComponent.Configuration) in EndpointCreator.cs
    NServiceBus.HostCreator.CreateWithInternallyManagedContainer(NServiceBus.EndpointConfiguration) in HostCreator.cs
    System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
    System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(System.Threading.Tasks.Task)
    System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(System.Threading.Tasks.Task)
    System.Runtime.CompilerServices.ConfiguredTaskAwaitable<TResult>.ConfiguredTaskAwaiter.GetResult()
    NServiceBus.Endpoint.Start(NServiceBus.EndpointConfiguration) in Endpoint.cs
    ...
    [Call Stack Truncated]

</details>

Reply:
nm, I had to disable the delayed recoverability.